src/twisted/web/error.py

```diff
@@ -22,22 +22,20 @@
 
 
 from collections.abc import Sequence
-from typing import cast
+from typing import Optional, Union, cast
 
 from twisted.python.compat import nativeString
 from twisted.web._responses import RESPONSES
 
 
-def _codeToMessage(code):
+def _codeToMessage(code: Union[int, bytes]) -> Optional[bytes]:
```

Comment:
this is private method...so we should be fine.

But maybe is better to be strict and only allow bytes as input?

Reply:
I did that in an earlier draft, but it led to awkward control flow to avoid coercing things that aren't already coerced (the `super()` arguments). I didn't think the compatibility implications of this worth the tidings of an internal interface.